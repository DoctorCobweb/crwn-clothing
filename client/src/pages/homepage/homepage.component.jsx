
import React from 'react';

import Directory from '../../components/directory/directory.component';

import { HomePageContainer } from './homepage.styles';

const HomePage = () => {
  // throw some erros to see how error boundaries are working
  // throw new Error('quafadf');
  // throw 'asdf';

  return (
  <HomePageContainer>
    <Directory />
  </HomePageContainer>
  );
};

export default HomePage;