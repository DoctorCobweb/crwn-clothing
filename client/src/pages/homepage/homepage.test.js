import React from 'react';
import { shallow } from 'enzyme';

import { HomePage } from './homepage.component';

describe('HomePage component', () => {
  it('should render HomePage', () => {
    expect(shallow(<HomePage />).debug()).toMatchSnapshot();
  });
});