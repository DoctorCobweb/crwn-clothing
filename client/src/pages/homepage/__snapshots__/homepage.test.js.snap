// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage component should render HomePage 1`] = `
"<HomePageContainer>
  <Connect(Directory) />
</HomePageContainer>"
`;
